src/counter.js

```diff
@@ -17,7 +17,7 @@ class Counter {
   /**
    * Update the counter by amount.
    * @param {number} [amount=1] amount to add to the counter.
-   * @return {undefined}
+   * @return {void}
```

Comment:
moved these to void to be in line with the types. Thinking about this a bit more, I don't think these doc comment changes are needed anymore. Just saw the error when I added tsd. But there are no errors on other methods that don't line up.